### DISCUSSION TITLE
Artist not beeing downloaded

### DISCUSSION THREAD

**Discussion by Makario1337** [RandomNinjaAtk/docker-lidarr-extended#32] [General]
I would like to grab releases from this Artist: https://musicbrainz.org/artist/e028fab5-39ae-4ed9-b8c2-c4344d88b171

The Problem is, that all Releases are found under Other + Audio drama.

Lidarr is not picking them up by its own. So i added the releases manually by copying the release id 
(eg. https://musicbrainz.org/release-group/19e6209b-2ddc-30b8-9273-484bd075fe7b and added it this way: 
Lidarr:19e6209b-2ddc-30b8-9273-484bd075fe7b)

However some worked, some did not. 
Is there any way to grab all Albums belonging to this Artist?

For further help: Albums 203-215 worked 216 and 217 are missing.

Thanks :)

**Comment by RandomNinjaAtk** [comment]
Script only searches for what Lidarr says to search for, so its likely an issue with Musicbrainz data for those albums.

If a release group on musicbrainz is considered unknown, Lidarr will never pull it in and the only way to fix it is to update musicbrainz with the correct info.

You also may want to check with Lidarr support groups to see if there is anything more specific about it because this sounds Lidarr related and not script related.
